src/main/java/org/folio/circulation/domain/notice/schedule/JsonScheduledNoticeMapper.java

```diff
@@ -72,7 +75,7 @@ public static JsonObject mapToJson(ScheduledNotice notice) {
       .put(LOAN_ID, notice.getLoanId())
       .put(REQUEST_ID, notice.getRequestId())
       .put(NEXT_RUN_TIME, notice.getNextRunTime().withZone(DateTimeZone.UTC).toString())
```

Comment:
This does not specify the format of the date and time output, is this the default?

This can be done using `JsonPropertyWriter.write()` instead.